File: frontend/portfolioERP-web/src/app/features/suppliers/models/update-supplier-request.ts
export interface UpdateSupplierRequest {
  supplierCode: string;
  companyName: string;
  contactName: string | null;
  vatNumber: string | null;
  taxCode: string | null;
  email: string;
  phone: string | null;
  address: string | null;
  city: string | null;
  province: string | null;
  postalCode: string | null;
  country: string;
  isActive: boolean;
}

File: backend/PortfolioERP.Application/Features/Suppliers/UpdateSupplierRequest.cs
namespace PortfolioERP.Application.Features.Suppliers;

public sealed record UpdateSupplierRequest(
	string SupplierCode,
	string CompanyName,
	string? ContactName,
	string? VatNumber,
	string? TaxCode,
	string Email,
	string? Phone,
	string? Address,
	string? City,
	string? Province,
	string? PostalCode,
	string Country,
	bool IsActive);